src/RoutingControllersOptions.ts

```diff
@@ -1,4 +1,5 @@
 import {ClassTransformOptions} from "class-transformer";
+import {TransformValdiationOptions} from "class-transformer-validator";
```

Comment:
You are right, I checked that now. There are two options:
- use `skipLibCheck` option for tsc
- as it was said before, we could make this libs the dependency
- make a JS require and forget about typings for this 😉